Backend/Meta-TV2-api/Meta-TV2-AccessLayer/Controllers/ApiController.cs

```diff
@@ -103,4 +108,30 @@ public class Slide : ControllerBase {
         var delete = await businessRules.ArchiveSlide(id);
         return delete ? Ok() : BadRequest($"Failed to remove slide: {id}");
     }
+}
+
+[Authorize(Roles = "Admin")]
+[Route("[Controller]")]
+public class Admin : ControllerBase {
+    IBusinessRules businessRules = new BusinessRules();
+    
+    [HttpPost("banUser")]
+    public async Task<IActionResult> BanUser(string alias) {
+        var created = await businessRules.BanUser(alias);
+        return created ? Ok() : BadRequest("Failed to ban user!");
+    }
+
+    [HttpGet]

```

Comment:
Potentially unclear name for the endpoint

Reply:
I don't quite see what point of it is unclear. It would currently be under "/Admin/banUser?alias=<obj>". However if you have a better name feel free to edit.

Reply:
GetAdminInformation is unclear, banUser is good. The endpoint for GetAdminInformation is just /Admin/

Reply:
Ah makes sense, I will do a fix!